students/WWhite/lesson09/oo_mailroom.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+# -------------------------------------#
+# Desc: Object Oriented Mailroom
+# Dev: Will White
+# Date: 7/2/2018
+# ChangeLog: (When,Who,What)
+# -------------------------------------#
+import math
+
+
+class Donor(object):
+    def __init__(self, name="", donation=0):
+        self.name = name
+        self.total_donation = 0
+        self.donationCount = 0
+
+        self.donation = donation
+
+    @property
+    def name(self):
+        return self._name
```

Comment:
couple things here:
* your class attributes should be defined in `__init__` (neither `self._name` or `self._donation` are)
* property is useful when attribute needs additional functionality or to ensure that the value cannot be changed. this is not the case here.